rma_account/views/account_move_view.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="invoice_form" model="ir.ui.view">
+        <field name="name">account.move.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <field name="name" position="before">
+                <div
+                    class="oe_button_box"
+                    attrs="{'invisible': [('rma_count', '=', 0)]}"
+                >
+                    <button
+                        type="object"
+                        name="action_view_rma_customer"
+                        class="oe_stat_button"
+                        icon="fa-eject"
+                        groups="rma.group_rma_customer_user,rma.group_rma_supplier_user"
+                    >
+                        <field name="rma_count" widget="statinfo" string="RMA" />
+                    </button>
+                </div>
+            </field>
+        </field>
+    </record>
+
+
+    <record id="view_invoice_line_form" model="ir.ui.view">
+        <field name="name">rma.invoice.line.form</field>
+        <field name="model">account.move.line</field>
+        <field name="inherit_id" ref="account.view_move_line_form" />
+        <field eval="16" name="priority" />
+        <field name="arch" type="xml">
+            <data>
+                <field name="name" position="after">
+                    <field name="rma_line_count" invisible="1" />
+                    <field name="rma_line_id" string="RMA line originated" />
+                    <notebook attrs="{'invisible': [('rma_line_count', '=', 0)]}">
+                        <page string="RMA Lines">
+                            <field name="rma_line_ids" />
+                        </page>
+                    </notebook>
+                </field>
+            </data>
+        </field>
+    </record>
+
+
+    <record id="view_invoice_customer_rma_form" model="ir.ui.view">
+        <field name="name">account.move.customer.rma</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <field name="invoice_vendor_bill_id" position="after">
+                 <field
+                    name="add_rma_line_id"
+                    context="{'rma': True}"
+                    domain="[('partner_id', '=', partner_id)]"
+                    attrs="{'readonly': [('state','not in',['draft'])],
+                         'invisible': ['|', ('state', '=', 'paid')]}"
```

Comment:
@MateuGForgeFlow You are missing an argument or you should remove the operator...